instrumentation/ratpack/ratpack-1.7/library/src/main/java/io/opentelemetry/instrumentation/ratpack/v1_7/OpenTelemetryExecInitializer.java

```diff
@@ -9,7 +9,7 @@
 import ratpack.exec.ExecInitializer;
 import ratpack.exec.Execution;
 
-final class OpenTelemetryExecInitializer implements ExecInitializer {
+public final class OpenTelemetryExecInitializer implements ExecInitializer {
```

Comment:
If you need to make this class `public` you should move it to `internal` package to avoid it being a part of the public api.